WalletWasabi.Packager/usermanuals/macsign.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+cd ~/Desktop/WalletWasabi/WalletWasabi.Packager
+git fetch
+git pull
```

Comment:
```suggestion
git pull
```

as `git pull` actually involves `git fetch` [as per](https://www.git-scm.com/docs/git-pull):

> More precisely, `git pull` runs `git fetch` with the given parameters and then depending on configuration options or command line flags, will call either `git rebase` or `git merge` to reconcile diverging branches.

Reply:
ACK